BudgetSquirrel.Api/ClientApp/src/app/budgets/budget-overview/budget-overview.component.html

```diff
@@ -1,11 +1,12 @@
 <div id="budgets">
 
   <!-- Was an error loading the budget-->
-  <div *ngIf="this.wasError">
-    <span>There was an error loading your budgets, please try again.</span>
-  </div>
-
-  <ng-template *ngIf="!this.wasError">
+  <ng-template [ngIf]="this.wasError" [ngIfElse]="notWasError">
```

Comment:
What is the difference between `wasError` and `notWasError`?

Reply:
lol the notWasError is the else if template of the wasError. I just called it notWasError to make it obviouse it's a straight negation. It's just the reference to a blank template.